code/modules/research/designs/autolathe_designs.dm

```diff
@@ -459,7 +459,7 @@
 	name = "E-revolver Charge Pack"
 	id = "e_charger"
 	build_type = AUTOLATHE
-	materials = list(MAT_METAL = 30000, MAT_GLASS = 6000)
+	materials = list(MAT_METAL = 20000, MAT_GLASS = 6000)
 	build_path = /obj/item/ammo_box/magazine/detective/speedcharger
```

Comment:
Probably gonna need to change the materials too to prevent infinite metal.
Please change the `materials = list(MAT_METAL = 30000)` variable on the speedcharger. Currently it costs 15 metal and 3 glass (unupgraded) to print, and returns 15 metal. But if you go through with this change, it will cost 10 metal and 3 glass, but return 15 metal

Reply:
Changed to turn into equivalent metal, without the glass confiscation.